src/index.html

```diff
@@ -3,11 +3,41 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__icon" src="images/imac.jpeg"
+       alt="icon">
+      <p class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__subtitle">Product code: 195434</p>
+      <div class="container-stars">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="rewiews">
+          <p>Reviews: 5</p>
+        </div>
+      </div>
+      <div class="price">
+        <div class="price-text">
+          <p>Price:</p>
+        </div>
+        <div class="price-coast">
+          <p>$2,199</p>
+        </div>
+      </div>
+      <a href="#" class="button-link"
+      data-qa="hover">buy</a>
```

Comment:
<img width="1051" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/64750958/253c39ea-c555-4436-9ce0-242e4781d8de">

Reply:
isn't fixed. Here is [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)